main.tf

```diff
@@ -0,0 +1,65 @@
+provider "aws" {
+  region = "eu-central-1"
```

Comment:
Pridal bych backend na s3, takhle by to neslo updatovat mimo toho, kdo to vyrobil:

```
provider "aws" {
   region = "eu-central-1"

   backend s3 {
    ...
   }
}
```